src/__tests__/app.integration.test.js

```diff
@@ -273,30 +273,29 @@ describe('Ensure health record outbound XML is unchanged', () => {
   });
 
   it('should verify that a fragment with external attachments is unchanged by xml changes', async () => {
-    // when
-    await createRegistrationRequest(
-      outboundConversationId,
-      ehrRequestMessageId,
-      nhsNumber,
-      odsCode
-    );
-
-    retrieveIdsFromEhrRepo.mockResolvedValueOnce(ehrRepoMessageIdResponse);
-
+    // given
     const fragmentMessageIds = Object.keys(originalFragments);
-    for (let messageId of fragmentMessageIds) {
-      // add records of the old message ids to database table
-      // new message ids are mostly same as the old ones, with last char replaced as '0', in order to guarantee the .sort() at expect statement give the same order.
-      await createMessageIdReplacement(messageId, messageId.slice(0, 35) + '0')
+    const messageIdReplacements = fragmentMessageIds.map(oldMessageId => {
+      return {
+        oldMessageId,
+        newMessageId: oldMessageId.slice(0, 35) + '0'
+      }
```

Comment:
Is there a blank line here or is GitHub just displaying it wrongly?

Reply:
GitHub glitch